services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from db.models import models

```

Comment:
The import statement `from db.models import models` is incorrect. It should be `from django.db import models` to use `models.QuerySet`.